### DISCUSSION TITLE
Shouldn't the results of 'npm outdated' be inverted?

### DISCUSSION THREAD

**Discussion by csvan** [npm/feedback#106] [CLI] [answered]
Maybe I am too used to yarn here, but currently the output of `npm outdated` has yellow for major versions, but red for minor versions. Shouldn't it be the other way around?

<img width="967" alt="Screenshot 2020-11-27 at 23 12 45" src="https://user-images.githubusercontent.com/1127729/100486918-7f493100-3106-11eb-8bb5-da58ddda03a7.png">

**Comment by ljharb** [comment]
A non-major version you’re behind means you might be missing security bugs, and you should be able to update to it safely.

A major version behind might mean you’re blocked from updating to it due to platform or feature support, or larger changes would be required to update.

In other words, the red ones are both urgent and theoretically trivial; the yellow ones may not even apply.

**Comment by MylesBorins** [comment, marked as answer]
Our team is working on design changes that will hopefully make the output of `npm outdated` easier to read. The colors are more about how important it is that you update a dependency, red needs action vs yellow is good to be aware of.

We are aware that this can be confusing and are going to look into improving this in the new year
